config.json

```diff
@@ -721,4 +901,4 @@
     "used_for/cross_platform_development",
     "used_for/scripts"
   ]
-}
```

Comment:
Was this trailing newline removed intentionally? POSIX lines should end in a newline character.